projects/rocprofiler-sdk/tests/rocprofv3/rocpd-scratch/CMakeLists.txt

```
@@ -0,0 +1,107 @@
+#
+# rocprofv3 rocpd tests
+#
+cmake_minimum_required(VERSION 3.21.0 FATAL_ERROR)
+
+project(
+    rocprofiler-sdk-tests-rocprofv3-rocpd-scratch
+    LANGUAGES CXX
+    VERSION 0.0.0)
+
+find_package(rocprofiler-sdk REQUIRED)
+
+set(rocprofv3-rocpd-scratch-env
+    "${ROCPROFILER_MEMCHECK_PRELOAD_ENV}"
+    "PYTHONPATH=${rocprofiler-sdk_LIB_DIR}/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages"
+    "OMPI_ALLOW_RUN_AS_ROOT=1"
+    "OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1")
+
+rocprofiler_configure_pytest_files(CONFIG pytest.ini COPY conftest.py validate.py)
+
+find_package(Python3 REQUIRED)
+
+#########################################################################################
+#
+# generate rocpd database and json for comparison
+#
+#########################################################################################
+
+add_test(
+    NAME rocprofv3-test-rocpd-scratch-execute
+    COMMAND
+        $<TARGET_FILE:rocprofiler-sdk::rocprofv3> --scratch-memory-trace -d
+        ${CMAKE_CURRENT_BINARY_DIR}/rocpd-input-data -o out --output-format rocpd json
+        --runtime-trace --kernel-rename --output-config -- $<TARGET_FILE:scratch-memory>)
```

```suggestion
        --runtime-trace --kernel-rename --output-config -- $<TARGET_FILE:scratch-memory>)
```
@mcrnobrn-amd do you really need the output-config json created for this test?  It's not used?  But you want to generate it for reference?  Just curious why you included it.

---

We don't need it. It was only for reference. I've removed it.